content/en/blog/_posts/2021-0420-cncf-dan.md

```diff
@@ -0,0 +1,196 @@
+
+<!-- NOTE:
+I am not sure where this anecdote fits into the story about Dan, Kubernetes, and ii.  
+In the interest of having a clear narrative, I edited it out.
+
+During this time Hippie was preparing a presentation about the work he was doing
+with the [Marine Reach](https://marinereach.com/) vessels. He asked Bob and his
+team if he could show them his presentation for some feedback from a live
+audience. The team really resonated with Hippie’s passion for the things he was
+involved in with Marine Reach. -->
+
+During the MTN project, Bob shipped servers to Hippie's home in Portland,
+Oregon, to help with the network-booting work. When Hippie later moved to
+Aotearoa New Zealand, he was unable to bring the servers with them. Hippie then
+asked bob if he had any money available to help relocate the servers to Aotearoa
+New Zealand. Bob introduced Hippie to someone who could help. This person was
+[Dan Kohn](https://en.wikipedia.org/wiki/Dan_Kohn). Bob introduced the two of
+them, telling Dan about Hippie's work and values. A relationship was formed, the
+servers were relocated, and are still used as part of the infrastructure for
+Hippie's vision.
+
+## Balena and Cross-Cloud
+
+Now in Aotearoa New Zealand, Hippie began doing work for [Balena](https://www.balena.io/)
+with [Denver Williams](https://github.com/denverwilliams). This technology
+enabled the "image pull and networking boot" in new, exciting ways. Inspired,
+Hippie set to use this technology with Kubernetes, by pulling down the images
+and installing [GitLab](https://about.gitlab.com) on top of it. He showed a demo
+of this to Dan who, impressed by this innovative approach to using Kubernetes,
+asked Hippie if he'd like to join the CNCF to help demonstrate what the
+orginzation is capable of. Hippie jumped in, helping to create a [Demo of CNCF
+technologies](https://github.com/cncf/demo/).
+
+While the demo did not get much traction, it was foundational in expoloring
+ideas which evolved into Hippie's next project with CNCF. Dan had a vision to
+get all cloud providers actively engaged in the cloud-native experience. This
+vision manifested as [Cross-Cloud](https://github.com/crosscloudci/cross-cloud),
+a way to concretely show the work of CNCF through a web
+frontend([cncf.ci](https://cncf.ci/)) that showed all the projects available on
+participating cloud providers. [Taylor Carpenter](https://github.com/taylor), a
+long time friend, continued the project with the [vulk.coop](https://vulk.coop)
+team and Hippie moved on to tackle new challenges with Dan.
+
+## The start of Conformance
+
+In 2017 Hippie attended his first CloudNativeCon + KubeCon Europe in Berlin,
+Germany. It was here that Dan introduced the Kubernetes conformance standard,
+and the Kubernetes Certified Service Provider program.
+
+A year later, in February 2018 [Kenichi Omichi](https://github.com/oomichi) dug
+into the Kubernetes logs and found that Kubernetes had 481 API endpoints only 53
```

Comment:
Probably a comma after "481 API endpoints"

Reply:
Yes, that would read better. Fixed